### DISCUSSION TITLE
Configuring `expect.closeTo.precision` for an entire project. Is it possible?

### DISCUSSION THREAD

**Discussion by heysokam** [vitest-dev/vitest#6821] [Q&A] [answered]
I have a project where I need `0.001` floating point precision everywhere
I know I could pass `3` to every call of `closeTo`, or write an .extend that does that, but...
is there an existing way to configure the value instead of having to hack around it?

**Comment by hi-ogawa** [comment, marked as answer]
Currently it's not possible and `precision = 2` is hard-coded in many places. Probably going with own matcher like `expect.myCloseTo(x)` with a different value is not that bad.

**Reply by heysokam** [reply]
ty :writing_hand:
